.github/workflows/deploy_doc.yml

```
@@ -0,0 +1,10 @@
+name: Deploy doc
+
+on:
+  push:
+      branches: [master,test_workflow]
+
+jobs:
+
+  call-workflow-in-another-repo:
```

Use a more descriptive name, related to the task, this is a rather generic one (possibly applying also to other workflows). Something like:
```suggestion
  deploy-docs:
```
that also matches the workflow name above.

Just to point it out, you don't have to add `qibo`, since it is already the scope of the repo (indeed the repo name is `qibo`, no need to repeat).